src/app/screens/Accounts/Detail/index.tsx

```diff
@@ -494,36 +465,30 @@ function AccountDetail() {
             </div>
             <MenuDivider />
             <div className="flex justify-between items-end">
-              <div className="w-7/12 flex items-center gap-2">
-                <TextField
-                  id="liquidPublicKey"
-                  label={t("liquid.public_key.label")}
-                  type="text"
-                  value={liquidPublicKey}
-                  disabled
-                  endAdornment={
-                    liquidPublicKey && (
-                      <InputCopyButton value={liquidPublicKey} />
-                    )
-                  }
-                />
-                {liquidPublicKey && liquidKeyOrigin !== "secret-key" && (
-                  <Badge
-                    label="imported"
-                    color="green-bitcoin"
-                    textColor="white"
-                  />
-                )}
+              <div className="w-7/12">
+                <p className="text-gray-900 dark:text-white font-medium">
+                  {t("liquid.enable.title")}
+                </p>
+                <p className="text-gray-500 text-sm dark:text-neutral-500">
+                  {t("liquid.enable.description")}
+                </p>
               </div>
 
-              <div className="w-1/5 flex-none">
-                <Link to="liquid">
-                  <Button
-                    label={t("advanced_settings.label")}
-                    primary
-                    fullWidth
-                  />
-                </Link>
+              <div className="w-1/5 flex align-middle justify-center">
+                <Toggle
+                  key="isLiquidEnable"
+                  checked={isLiquidEnable || false}
+                  onChange={() => {
+                    setLiquidEnable(!isLiquidEnable);
+                  }}
+                />
+                <span className="ml-4">
```

Comment:
Normally we only have the toggle, not the text. And the toggle should be right aligned and centered vertically to match other toggles in the settings page (I can update this)

Reply:
@louisinger looks great, thanks!